src/layer/x86/x86_activation.h

```diff
@@ -17,34 +17,57 @@
 
 #include <math.h>
 #include "mat.h"
+#include "x86_usability.h"
 
-static inline float activation_ss(float v, int activation_type, const ncnn::Mat& activation_params)
+static forceinline float activation_ss(float v, int activation_type, const ncnn::Mat& activation_params)
 {
-    if (activation_type == 1)
+    switch (activation_type)
+    {
+    case 1:
     {
         v = fmax(v, 0.f);
+        break;
     }
-    else if (activation_type == 2)
+    case 2:
     {
         float slope = activation_params[0];
         v = v > 0.f ? v : v * slope;
+        break;
     }
-    else if (activation_type == 3)
+    case 3:
     {
         float min = activation_params[0];
         float max = activation_params[1];
         if (v < min)
             v = min;
         if (v > max)
             v = max;
+        break;
     }
-    else if (activation_type == 4)
+    case 4:
     {
         v = 1.f / (1.f + exp(-v));
+        break;
     }
-    else if (activation_type == 5)
+    case 5:
     {
         v = v * tanh(log(exp(v) + 1.f));
+        break;
+    }
+    case 6:
+    {
+        float alpha = activation_params[0];
+        float beta = activation_params[1];
+        float lower = -beta / alpha;
+        float upper = (1.f / alpha) + lower;
+        if (v < lower)
+            v = 0.f;
+        else if (v > upper)
+            ;
+        else
+            v = v * (v * alpha + beta);
+        break;
+    }
     }
```

Comment:
This code snippet is duplicated in most cpu-arch-specific implementations and in some naive layer code.
Create a new file named ```fused_activation.h``` in src/layers, and use it for much better code maintenance applicant.